docs/source/managed-federation/deployment.mdx

```diff
@@ -4,42 +4,28 @@ description: Best practices
+When rolling out changes to a [subgraph](../implementing-services/), we recommend the following workflow:
 
-1. Confirm the backward compatibility of each change by [running `apollo service:check`](./federated-schema-checks/) in your CI pipeline.
+1. Confirm the backward compatibility of each change by [running `rover subgraph check`](https://www.apollographql.com/docs/rover/subgraphs/#checking-subgraph-schema-changes) in your CI pipeline.
 2. Merge backward compatible changes that successfully pass schema checks.
-3. Deploy changes to the implementing service in your infrastructure.
+3. Deploy changes to the subgraph in your infrastructure.
 4. Wait until all replicas finish deploying.
-5. Call `apollo service:push` to update your managed federation configuration:
+5. Run [`rover subgraph publish`](https://www.apollographql.com/docs/rover/subgraphs/#publishing-a-subgraph-schema-to-apollo-studio) to update your managed federation configuration:
 
-    ```
-    ~$ apollo service:push --serviceName="launches" \
-                          --serviceURL="http://launches-graphql.svc.cluster.local:4001/" \
-                          --endpoint="http://localhost:4001/"
-
-    ✔ Loading Apollo Project
-    ✔ Uploading service to Apollo
-
-    The 'registry' service for the 'space-explorer@current' graph was updated
-
-    The gateway for the 'space-explorer@current' graph was updated with a new schema, composed from the updated 'launches' service
-
-    id      graph             variant
-    ──────  ────────────────  ───────
-    az329e  space-explorer    current
-    ```
-
-> **What's the difference between the `endpoint` and `serviceURL` options?** 
->
->The `endpoint` option sets the endpoint where your service's schema is fetched from during composition, whereas `serviceURL` sets the URL that the gateway communicates with at runtime. This separation is useful because implementing services **should not be publicly accessible**, so the `endpoint` might point to a locally running server or a file, whereas the `serviceURL` should be a URL accessible to the gateway.
+```bash
+rover subgraph publish my-supergraph@my-variant \
+  --schema ./accounts/schema.graphql \
+  --name accounts \
+  --routing-url https://my-running-subgraph.com/api
+```
 
 ## Pushing configuration updates safely
 
-Whenever possible, you should update your service configuration in a way that is backward compatible to avoid downtime. The best way to do this is to minimize the number of [breaking changes](https://www.apollographql.com/docs/studio/schema-checks/#potentially-breaking-changes) you make to your schemas.
+Whenever possible, you should update your subgraph configuration in a way that is backward compatible to avoid downtime. The best way to do this is to minimize the number of [breaking changes](https://www.apollographql.com/docs/studio/schema-checks/#potentially-breaking-changes) you make to your schemas.
```

Comment:
Suggestion: add something like, "As mentioned above, always run a check before publishing, and also..." -- something where if I didn't read the stuff about checks above I'd still understand that I'm not on my own to ensure no breaking changes but have a way to check